src/components/accessories/patientNewBill/styles.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../../styles/variables";
+@import "../../../../node_modules/susy/sass/susy";
+
+.patientNewBill {
+  display: flex;
+  flex-flow: column;
+  border: 1px solid hsl(2, 98%, 53%, .075);
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px hsl(2, 98%, 53%, .075);
+  padding: 16px 8px 16px 8px;
+  row-gap: 1em;
+  & > div:first-child {
+    display: flex;
+    column-gap: 1em;
+    @include susy-media($smartphone) {
+      row-gap: 2em;
+      flex-flow: column; 
+      align-items: center; 
+    }
+    & > .patientNewBill_left {
+      display: flex;
+      flex: 9;
+      justify-content: space-between;
+      flex-flow: column;
+      row-gap: 1em;
+      background-color: white;
+      & > .patientNewBill_buttons {
+        display: flex;
+        column-gap: 2em;
+        justify-content: space-between;
+      }
+      .addpayment {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        color: gray;
+      }
+      @include susy-media($smartphone) {
+        width: 1fr;
+      }
+    }
+    & > .patientNewBill_right {
+      display: flex;
+      flex-flow: column;
+      align-items: center;
+      background-color: hsl(2, 98%, 53%, .3);
+      border-radius: 8px;
+      padding: 1.5em .25em 1.5em .25em;
+      row-gap: 1em;
+      .smallButton, .textButton {
+        width: 10em;
+        margin: 0px;
+      }
+      .textButton {
+        background-color: rgba(75, 30, 29, 0.15) !important;
+      }
+      .two-row {
+        display: flex;
+        width:  9em;
+        justify-content: space-between;
+        & > span:first-child {
+          font-weight: 200;
+        }
+        & > span:last-child {
+          font-size: 1.25em;
+        }
+      }
+      .xs-divider, .lg-divider {
+        height: 1px;
+        background-color: black;
+      }
+      .xs-divider {
+        width: 3.75em;
+      }
+      .lg-divider {
+        width: 11.5em;
+      }
```

Comment:
It's not good for the responsiveness the usage of static width/height like this. There is no need to do that if you did use the flex rightly inside the parent element.